src/Sets/SparsePolynomialZonotope/merge_id.jl

```diff
@@ -0,0 +1,66 @@
+"""
+    merge_id(id1::AbstractVector{Int}, id2::AbstractVector{Int}, 
+             E₁::AbstractMatrix{N}, E₂::AbstractMatrix{N}) where {N}
+
+Given two `indexvector`s and their corresponding exponent matrices `E₁` and `E₂`, 
+`merge_id` brings the exponent matrices and id lists into a common format.
+
+### Inputs
+
+- `id1::AbstractVector{Int}`: Identifiers corresponding to the rows of `E₁`
+- `id2::AbstractVector{Int}`: Identifiers corresponding to the rows of `E₂`
+- `E₁::AbstractMatrix{N}`: First exponent matrix of size `(p₁ × h₁)`.
+- `E₂::AbstractMatrix{N}`: Second exponent matrix of size `(p₂ × h₂)`.
+
+### Outputs
+
+- `Ē₁::Matrix{N}`: Aligned version of `E₁` 
+- `Ē₂::Matrix{N}`: Aligned version of `E₂` 
+- `idx::Vector{Int}`: Merged identifier vector, containing all elements of `id1` and any new ones from `id2`.
+### Algorithm
+
+This method implements [KochdumperA21; Proposition 1](@citet).
+
+### Example
+
+```jldoctest
+julia> id1 = [1, 2];
+
+julia> E₁ = [1 2; 1 0];
+
+julia> id2 = [2, 3];
+
+julia> E₂ = [1 0 1; 3 2 0];
+
+julia> Ē₁, Ē₂, idx = merge_id(id1, id2, E₁, E₂)
+([1 2; 1 0; 0 0], [0 0 0; 1 0 1; 3 2 0], [1, 2, 3])
+```
+"""
+function merge_id(id1::AbstractVector{Int}, id2::AbstractVector{Int}, E₁::AbstractMatrix{N},
+                  E₂::AbstractMatrix{N}) where {N}
```

Comment:
```suggestion
                  E₂::AbstractMatrix{N}) where {N<:Integer}
```
Not really needed, but could catch potential misuse. You may ignore this one.